### PR TITLE
Add another Nim book to this list.

### PR DESCRIPTION
**Pull request by pyautogui**
Add the Computer Programming with the Nim Programming Language book to this list.

## What does this PR do?
Add resources

### Description
A comprehensive guide for people new to Nim.
### Why is this valuable (or not)?
It is one of the most complete Nim programming books.
### How do we know it's really free?
The link is the front page with the index.
### For book lists, is it a book? For course lists, is it a course? etc.
It is a book.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: books/free-programming-books.md [hunk 1]
```diff
@@ -2209,7 +2209,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Nim Basics](https://narimiran.github.io/nim-basics) - narimiran
 * [Nim by Example](https://nim-by-example.github.io) - Flaviu Tamas
 * [Nim Days](https://xmonader.github.io/nimdays) - Ahmed Thabet
-
+* [Computer Programming with the Nim Programming Language](http://ssalewski.de/nimprogramming.html) - Dr. Stefan Salewski
```
Comment (davorpa) [inline books/free-programming-books.md:2212]:
```suggestion

```

Revert to ensure alphabetical sort

File: books/free-programming-books.md [hunk 2]
```diff
@@ -2209,7 +2209,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Nim Basics](https://narimiran.github.io/nim-basics) - narimiran
```
Comment (davorpa) [inline books/free-programming-books.md:2209]:
```suggestion
* [Computer Programming with the Nim Programming Language](http://ssalewski.de/nimprogramming.html) - Dr. Stefan Salewski
* [Nim Basics](https://narimiran.github.io/nim-basics) - narimiran
```

Ensure alphabetical order

**Review by davorpa — CHANGES_REQUESTED** [review]
Hi @pyautogui 

Acording to guidelines, ensure that links guard alphabetical order. I suggest you the inline changes 😉

**Comment by eshellman** [comment]
super addition, but you're adding it twice.
Be sure to check the output of Travis-CI for linter errors. Alphabetical order and spacing are the most common problems.

**Review by davorpa — CHANGES_REQUESTED** [review]
Pay attention at suggested changes to revert line 2213 and then resolve CI/CD errors

**Review by davorpa — APPROVED** [review]
Supe :rocket:

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**